thesis/front/abstract.tex

```diff
@@ -32,43 +32,41 @@
     With the advance of science and technology,
 the computing power of smart devices has increased,
 and the size getting more compact,
-these make people easier to record sounds.
-However, as the recording equipment becomes easier to carry with,
-it also makes eavesdropping becomes more difficult to prevent.
+these make peoples easier to record sounds.
+However, as the recording equipment becomes easier to carry,
+eavesdropping becomes more challenging to prevent.
 If a private conversation is illegally recorded or captured,
-personal privacy may be seriously violated.
+it may be seriously violating our privacy.
 The development of science and technology must take account of privacy and convenience.
-Therefore, as people emphasis on personal privacy,
-how to prevent unauthorized recording is bound to be an issue that needs to be taken seriously.
-In summary, how to establish a privacy-guaranteed environment
-to avoid being eavesdropped or unauthorized recorded
+Therefore, as people emphasize personal privacy,
+preventing unauthorized recording is bound to be an issue that needs to take seriously.
+In summary, establishing a privacy-guaranteed environment
+to avoid being eavesdropped on or unauthorized recorded
 will be the challenge and goal of modern technology.
 
     Recent works have demonstrated that the non-linear impulse response of ultrasonic
-has ability to interferes with the microphone while recording.
+has the ability to interferes with the microphone while recording.
 When the signal-to-noise ratio between the original recording
 and the interference noise reaches a certain level,
-this is equivalent to a denial of service (DoS) attack on the microphone's recording,
-and the impact on the recording result is devastating.
-If an ultrasonic jammer is deployed in the chamber of private conversation,
-this can solve the problem of eavesdropping or unauthorized recording,
-but it also affect all microphones in the same area. For example,
+this is equivalent to a denial of service (DoS) attack on the microphone's recording. The impact on the recording result is devastating.
+Suppose deploy an ultrasonic jammer in the chamber of private conversation. In that case,
+this can solve eavesdropping or unauthorized recording,
+but it also affects all microphones in the same area. For example,
 if you want to record a secret meeting as evidence,
 or you want to create a verbatim script based on the recording after the meeting.
-Thus it's unable to achieve because the interference of ultrasonic microphone jammer.
+Thus it's unable to achieve because of the interference of the ultrasonic microphone jammer.
```

Comment:
weird, may need to reconstruct this sentence

Reply:
keep origin